docker/Dockerfile.helper

```diff
@@ -0,0 +1,25 @@
+# =============
```

Comment:
Is this comment correct, is the file really generated from templating?

Reply:
removed